detect-python.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Please more clearly comment and document the purpose of this script, when it's being used, by whom and why, and what it does.